guides/release/tutorial/ember-data.md

```diff
@@ -25,29 +25,31 @@ When we open the model file, we can see a blank class extending [`DS.Model`](htt
 
 ```javascript {data-filename=app/models/rental.js}
 import DS from 'ember-data';
+const { Model } = DS;
 
-export default DS.Model.extend({
+export default class RentalModel extends Model {
 
-});
+}
 ```
 
 Let's define the structure of a rental object using the same attributes for our rental that we [previously used](../model-hook/) in our hard-coded array of JavaScript objects -
 _title_, _owner_, _city_, _category_, _image_, _bedrooms_ and _description_.
 Define attributes by giving them the result of the function [`DS.attr()`](https://www.emberjs.com/api/ember-data/release/classes/DS/methods/attr?anchor=attr).
```

Comment:
Can you update the prose to check for other references to the method `DS.attr()`?

Reply:
@jenweber ah, so update it to `attr()`?

Reply:
I updated each reference to `attr` as well as other references using `DS.something`